src/structuralFunctionality/telemetry/telemetry.ts

```diff
@@ -289,23 +288,23 @@ export default class Telemetry extends Accessor {
     }
     private async _loadGoogleAnalytics(settings: any) {
 
-        return create(async (resolve, reject) => {
+        return new Promise(async (resolve, reject) => {
             const { googleAnalyticsKey } = settings?.config;
 
             const enableGoogle = this._isGoogleEnabled(settings);
             const scriptsExist = this._googleScriptsExist();
 
             if (!enableGoogle && scriptsExist) {
-                window[`ga-disable-${googleAnalyticsKey}`] = false; 
-                resolve();
+                window[`ga-disable-${googleAnalyticsKey}`] = false;
+                Promise.resolve();
```

Comment:
I believe this should just be `resolve()` cause that function is defined in the callback